sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -8,9 +8,10 @@
   "/ENTRY[entry]/experiment_institution": "Deutsches Elektronen-Synchrotron DESY",
   "/ENTRY[entry]/experiment_facility": "Free-Electron Laser FLASH",
   "/ENTRY[entry]/experiment_laboratory": "@attrs:metadata/creationLocation",
-  "/ENTRY[entry]/entry_identifier": "@attrs:metadata/_id",
+  "/ENTRY[entry]/entry_identifier": "@attrs:metadata/pid",
   "/ENTRY[entry]/USER[user]": {
     "name": "@attrs:metadata/principalInvestigator",
+    "affiliation": "t.b.d.",
     "role": "Principal investigator",
     "email": "@attrs:metadata/contactEmail"
```

Comment:
This contact email appears to be from the Data Owner, and not from the Principal Investigator. Is his/her email available?

Reply:
Same as before - in the current version of SciCat they are cutting a lot of info from beamtime metadata and not porting them into SciCat. This should be changed in the new version. At least the leader and PI will have their own affiliations and contact details.

Reply:
Then add manually for now.

Reply:
Directly in the beamtime folder we have access to the beamtime_metadata.json file which is used to populate some of SciCat fields and in principle we can read that file and use that info, but then some fields will be duplicated several times.

Reply:
How about you only take the fields that are not in SciCat from there?

Reply:
I put those information into the notebook for now.